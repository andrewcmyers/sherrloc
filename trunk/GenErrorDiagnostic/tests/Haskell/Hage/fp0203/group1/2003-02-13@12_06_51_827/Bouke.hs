module Bouke where

slaplat :: [[Int]] -> [Int]
slaplat [] = []
slaplat n  = head n ++ slaplat (n-1)

-- (n-1) should be (tail n)
-- 5,32-36 
