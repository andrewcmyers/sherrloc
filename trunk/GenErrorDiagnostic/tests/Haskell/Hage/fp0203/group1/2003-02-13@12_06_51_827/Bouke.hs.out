Compiling Bouke.hs
(5,34)-(5,35): Type error in infix application
 expression       : n - 1
 operator         : -
   type           : a       -> a   -> a      
   does not match : [[Int]] -> Int -> [[Int]]

Compilation failed with 1 error
